src/webgpu/api/validation/queue/destroyed/query_set.spec.ts

```diff
@@ -21,6 +21,15 @@ export const enum EncoderType {
 
 export const g = makeTestGroup(ValidationTest);
 
+g.test('beginRenderPass')
+  .desc(
+    `
+Tests that use a destroyed query set in beginRenderPass for occlusion query.
```

Comment:
Should note here that the render pass can be empty, no begin/end query calls are needed to test this.

Reply:
I see. That's a little surprising but I guess it makes sense. Would you mind filing a spec issue so editors can make sure that's intended behavior?

Reply:
Sorry, I realized this is probably in https://github.com/gpuweb/gpuweb/pull/1494 which I haven't gotten a chance to  review yet.

Reply:
Is this actually covered by that PR? I couldn't find it. If it's not, could you make a PR or open an issue on the spec?

Reply:
I made a separate PR for it (https://github.com/gpuweb/gpuweb/pull/1531) which also contains the query set state at its creation and destory.